3CodeChallenge/app/app/src/main/java/com/example/myapplication/Challenge4/StudentViewHolder.java

```diff
@@ -0,0 +1,35 @@
+package com.example.myapplication.Challenge4;
+
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.myapplication.R;
+
+public class StudentViewHolder extends RecyclerView.ViewHolder {
+
+    private TextView firstNameTextView, lastNameTextView;
+    private LinearLayout recyclerViewLinearLayout;
+
+    public StudentViewHolder(@NonNull View itemView) {
+        super(itemView);
+        firstNameTextView = itemView.findViewById(R.id.firstNameTextView);
+        lastNameTextView = itemView.findViewById(R.id.lastNameTextView);
+        recyclerViewLinearLayout = itemView.findViewById(R.id.recyclerViewLinearLayout);
```

Comment:
Please consider using a better name for linear layout. Remove the recycleView words from the name. It is confusing